pkg/driver/trace_driver.go

```diff
@@ -184,6 +193,22 @@ type InvocationMetadata struct {
 	ReadOpenWhiskMetadata *sync.Mutex
 }
 
+type InvocationMetadataDAG struct {
+	Function              []*common.Function
```

Comment:
This is the only field that is different from InvocationMetadata (Apart from deleted unused field). Can you reuse the original one?